src/hooks/UrqlUseSubscriptionWithHandler.re

```diff
@@ -48,25 +52,28 @@ let useSubscriptionResponseToRecordWithHandler =
   {fetching, data, error, response};
 };
 
-let useSubscriptionWithHandler =
+let useSubscription =
     (
-      ~request: UrqlTypes.request('response),
-      ~handler: UrqlTypes.parsedHandler('acc, 'response),
-    ) => {
+      type acc, type resp, type ret,
+      ~handler: handler(acc, resp, ret),
+      request: UrqlTypes.request(resp),
```

Comment:
I think I'd still like to keep `request` as a labelled argument here if possible. It feels like a nicer API to have two labelled args rather than one labelled and one unlabelled. I'm forgetting, is there a specific reason we can't do this?

Reply:
There's no issue in having only labeled arguments, only if all the labeled arguments are optional with `=?` it's required to have at least one non optional argument.